### PR TITLE
Update CNA tests more

### PR DESCRIPTION
**Pull request by ijjk**
Follow-up to https://github.com/vercel/next.js/pull/57025 this updates other places we run the `create-next-app` CLI in tests.

### CODE REVIEW

**Review by huozhi — DISMISSED** [review]

**Comment by ijjk** [comment]
## Failing test suites 
Commit: d94c57594bf8d86ab3ef24f66edc9e158f116a2d

`pnpm test test/integration/css-features/test/index.test.js` 
- Custom Properties: Pass-Through Modern > production mode > should have compiled successfully

<details>
<summary>Expand output</summary>

  ● Custom Properties: Pass-Through Modern › production mode › should have compiled successfully

    expect(received).toBe(expected) // Object.is equality

    Expected: 0
    Received: 1

      55 |
      56 |     it('should have compiled successfully', () => {
    > 57 |       expect(code).toBe(0)
         |                    ^
      58 |       expect(stdout).toMatch(/Compiled successfully/)
      59 |     })
      60 |

      at Object.toBe (integration/css-features/test/index.test.js:57:20)

</details>


Read more about building and testing Next.js in [contributing.md](https://github.com/vercel/next.js/blob/canary/contributing.md).

**Comment by ijjk** [comment]
# Stats from current PR

<details open>
<summary><strong>Default Build</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/other-tests | Change |
| - | - | - | - |
| buildDuration | 10.4s | 10.4s | N/A |
| buildDurationCached | 6.2s | 6s | N/A |
| nodeModulesSize | 174 MB | 174 MB | ✓ |
| nextStartRea..uration (ms) | 519ms | 527ms | N/A |

</details>

<details>
<summary><strong>Client Bundles (main, webpack)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/other-tests | Change |
| - | - | - | - |
| 199-HASH.js gzip | 27.5 kB | 27.5 kB | ✓ |
| 3f784ff6-HASH.js gzip | 53.3 kB | 53.3 kB | ✓ |
| 99.HASH.js gzip | 182 B | 182 B | ✓ |
| framework-HASH.js gzip | 45.3 kB | 45.3 kB | ✓ |
| main-app-HASH.js gzip | 254 B | 252 B | N/A |
| main-HASH.js gzip | 32.9 kB | 32.9 kB | N/A |
| webpack-HASH.js gzip | 1.75 kB | 1.75 kB | N/A |
| Overall change | 126 kB | 126 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/other-tests | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/other-tests | Change |
| - | - | - | - |
| _app-HASH.js gzip | 206 B | 205 B | N/A |
| _error-HASH.js gzip | 182 B | 180 B | N/A |
| amp-HASH.js gzip | 506 B | 505 B | N/A |
| css-HASH.js gzip | 322 B | 323 B | N/A |
| dynamic-HASH.js gzip | 2.57 kB | 2.57 kB | N/A |
| edge-ssr-HASH.js gzip | 260 B | 259 B | N/A |
| head-HASH.js gzip | 350 B | 350 B | ✓ |
| hooks-HASH.js gzip | 369 B | 369 B | ✓ |
| image-HASH.js gzip | 4.36 kB | 4.36 kB | N/A |
| index-HASH.js gzip | 256 B | 256 B | ✓ |
| link-HASH.js gzip | 2.64 kB | 2.63 kB | N/A |
| routerDirect..HASH.js gzip | 312 B | 311 B | N/A |
| script-HASH.js gzip | 385 B | 384 B | N/A |
| withRouter-HASH.js gzip | 307 B | 308 B | N/A |
| 1afbb74e6ecf..834.css gzip | 106 B | 106 B | ✓ |
| Overall change | 1.08 kB | 1.08 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/other-tests | Change |
| - | - | - | - |
| _buildManifest.js gzip | 484 B | 482 B | N/A |
| Overall change | 0 B | 0 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/other-tests | Change |
| - | - | - | - |
| index.html gzip | 528 B | 529 B | N/A |
| link.html gzip | 541 B | 541 B | ✓ |
| withRouter.html gzip | 523 B | 524 B | N/A |
| Overall change | 541 B | 541 B | ✓ |

</details>

<details>
<summary><strong>Edge SSR bundle Size</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/other-tests | Change |
| - | - | - | - |
| edge-ssr.js gzip | 93.7 kB | 93.7 kB | N/A |
| page.js gzip | 155 kB | 155 kB | N/A |
| Overall change | 0 B | 0 B | ✓ |

</details>

<details>
<summary><strong>Middleware size</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/other-tests | Change |
| - | - | - | - |
| middleware-b..fest.js gzip | 622 B | 624 B | N/A |
| middleware-r..fest.js gzip | 150 B | 151 B | N/A |
| middleware.js gzip | 22.5 kB | 22.5 kB | N/A |
| edge-runtime..pack.js gzip | 1.92 kB | 1.92 kB | ✓ |
| Overall change | 1.92 kB | 1.92 kB | ✓ |

</details>

</details>
Commit: d94c57594bf8d86ab3ef24f66edc9e158f116a2d

File: packages/next/src/build/collect-build-traces.ts [hunk 1]
```diff
@@ -406,11 +406,9 @@ export async function collectBuildTraces({
               return await fs.readFile(p, 'utf8')
             } catch (e) {
               if (isError(e) && (e.code === 'ENOENT' || e.code === 'EISDIR')) {
-                // handle temporary internal webpack files
                 if (p.match(/static[/\\]media/)) {
                   return ''
                 }
-                return null
```
Comment (styfle) [inline packages/next/src/build/collect-build-traces.ts:413]: Did you intend to change this line?

File: test/integration/create-next-app/templates-app.test.ts [hunk 2]
```diff
@@ -13,12 +13,10 @@ import {
   shouldBeTemplateProject,
   spawnExitPromise,
 } from './lib/utils'
-//import { Span } from 'next/dist/trace'
 
 import { useTempDir } from '../../lib/use-temp-dir'
 import { fetchViaHTTP, findPort, killApp, launchApp } from 'next-test-utils'
 import resolveFrom from 'resolve-from'
-//import { createNextInstall } from '../../lib/create-next-install'
```
Comment (styfle) [inline test/integration/create-next-app/templates-app.test.ts:21]: Similarly here - was this intended?

File: packages/next/src/build/collect-build-traces.ts (hunk 1 above)
Comment (ijjk) [reply packages/next/src/build/collect-build-traces.ts:413]: Yeah attempting to fix flake with ENOENT can move to a separate PR

File: test/integration/create-next-app/templates-app.test.ts (hunk 2 above)
Comment (ijjk) [reply test/integration/create-next-app/templates-app.test.ts:21]: Yeah no need to keep the comment around as we have the TODO

**Review by huozhi — APPROVED** [review]

File: packages/next/src/build/collect-build-traces.ts (hunk 1 above)
Comment (ijjk) [reply packages/next/src/build/collect-build-traces.ts]: Moved to https://github.com/vercel/next.js/pull/57092

**Pull request merged**